cycode/cyclient/scan_config_base.py

```diff
@@ -38,9 +48,14 @@ def get_service_name(self, scan_type: str) -> str:
     def get_detections_prefix(self) -> str:
         return '5016'
 
+    def get_scan_service_prefix(self) -> str:
+        return '5004'
+
 
 class DefaultScanConfig(ScanConfigBase):
-    def get_service_name(self, scan_type: str) -> str:
+    def get_service_name(self, scan_type: str, should_use_scan_service: bool = False) -> str:
+        if should_use_scan_service:
+            return self.get_scan_service_prefix()
```

Comment:
if we want to have `get_scan_service_prefix` instead of literal pls replace the literal below (for sca and sast)

but imo as we already have literals for secrets and iac detector ig we are fine without additional abstract method here 

i mean it's duplication in both cases. for dev and default configs. so it could be more consistent without additional method